### ISSUE TITLE
libevdev is missing from the toolchain

### ISSUE THREAD

**Issue by matteodelabre** [toltec-dev/toolchain#6]
Thanks for having a look, indeed `libevdev` seems to be missing from the base toolchain. It is included on the reMarkable though, so this seems like a toolchain error.

_Originally posted by @timower in https://github.com/toltec-dev/toltec/issues/261#issuecomment-767096919_

**Comment by matteodelabre** [comment]
Upstream is at https://www.freedesktop.org/wiki/Software/libevdev/

**Comment by matteodelabre** [comment]
Fixed in release v1.3.

**Comment by not-a-doctor-stromberg** [comment]
Did this regress in 2.1?

```
me@Aarons-Air rM2-stuff % docker run -it --rm --network host -v`pwd`:/src ghcr.io/toltec-dev/base:v2.1 bash
root@docker-desktop:/# ldconfig -p | grep evdev
root@docker-desktop:/# apt-get update
Get:1 http://deb.debian.org/debian unstable InRelease [165 kB]
Get:2 http://deb.debian.org/debian unstable/main amd64 Packages [8780 kB]
Fetched 8945 kB in 2s (3823 kB/s)
Reading package lists... Done
root@docker-desktop:/# apt-get install libevdev-dev
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  libevdev2
Suggested packages:
  libevdev-doc
The following NEW packages will be installed:
  libevdev-dev libevdev2
0 upgraded, 2 newly installed, 0 to remove and 132 not upgraded.
Need to get 85.5 kB of archives.
After this operation, 432 kB of additional disk space will be used.
Do you want to continue? [Y/n] Y
Get:1 http://deb.debian.org/debian unstable/main amd64 libevdev2 amd64 1.11.0+dfsg-1 [34.0 kB]
Get:2 http://deb.debian.org/debian unstable/main amd64 libevdev-dev amd64 1.11.0+dfsg-1 [51.5 kB]
Fetched 85.5 kB in 0s (195 kB/s)   
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package libevdev2:amd64.
(Reading database ... 18028 files and directories currently installed.)
Preparing to unpack .../libevdev2_1.11.0+dfsg-1_amd64.deb ...
Unpacking libevdev2:amd64 (1.11.0+dfsg-1) ...
Selecting previously unselected package libevdev-dev:amd64.
Preparing to unpack .../libevdev-dev_1.11.0+dfsg-1_amd64.deb ...
Unpacking libevdev-dev:amd64 (1.11.0+dfsg-1) ...
Setting up libevdev2:amd64 (1.11.0+dfsg-1) ...
Setting up libevdev-dev:amd64 (1.11.0+dfsg-1) ...
Processing triggers for libc-bin (2.31-10) ...
root@docker-desktop:/# ldconfig -p | grep evdev
	libevdev.so.2 (libc6,x86-64) => /usr/lib/x86_64-linux-gnu/libevdev.so.2
	libevdev.so (libc6,x86-64) => /usr/lib/x86_64-linux-gnu/libevdev.so
```
